docker/wc-admin-local-dev/README.md

```diff
@@ -0,0 +1,46 @@
+# WooCommerce Local Docker Setup
+
+This is PHP + Nginx + Mysql Docker setup for local WooCommerce Admin development.
+
+You should have the latest Docekr and NPM installed to get started.
+
+## Getting Started
+
+1. From the root of WC Admin directory, run `npm run docker-dev:config`. You can choose Nginx and Mysql port during the config process.
+2. Run `npm run docker-dev:up`. This will run the docker containers.
+3. Run `npm run docker-dev:setup`. This will run npm install, composer install, and the initial build process.
+4. Step #3 might take over 10+ minutes if it's your first time running it. Grab a cup of coffee :coffee: 
```

Comment:
Phew 10 minutes? That seems extreme. The docker setup I built here: https://github.com/samueljseay/wp-docker-scripts takes 2 mins from cold, I'm just wondering if maybe you're downloading a huge image here or something?